localstack-core/localstack/services/events/utils.py

```diff
@@ -42,6 +49,23 @@ def to_json_str(obj: Any, separators: Optional[tuple[str, str]] = (",", ":")) ->
     return json.dumps(obj, cls=EventJSONEncoder, separators=separators)
 
 
+def extract_region_and_account_id(
+    name_or_arn: EventBusNameOrArn, context: RequestContext
+) -> tuple[str, str]:
+    """Returns the region and account id from the arn,
+    or falls back on the region and account id of the context"""
+    account_id = None
+    region = None
+    if ArnParser.is_arn(name_or_arn):
+        parsed_arn = parse_arn(name_or_arn)
+        region = parsed_arn.get("region")
+        account_id = parsed_arn.get("account")
+    if not account_id or not region:
+        region = context.get("region")
```

Comment:
Can region and account_id in context be None? (typing `str` or `str | None`)

Reply:
yes if it is not ARN arn that is provided but a name